integrations/terraform/testlib/autoupdate_version_test.go

```diff
@@ -0,0 +1,131 @@
+/*
+Copyright 2015-2021 Gravitational, Inc.
```

Comment:
```suggestion
Copyright 2025 Gravitational, Inc.
```

I don't know if this is important, honestly